services/provider/server/storageclaim_test.go

```diff
@@ -22,14 +22,14 @@ func Test_getStorageClassClaimName(t *testing.T) {
 				consumerUUID:          "consumer-uuid",
 				storageClassClaimName: "storage-class-claim-name",
 			},
-			want: "storageclassclaim-eea7000672540cd95e00f454f8f73e74",
+			want: "storageclassrequest-eea7000672540cd95e00f454f8f73e74",
```

Comment:
replace variable name with storageRequestName at line 10 and file need to be renamed to storagerequired_test